### DISCUSSION TITLE
Would like to help with the Discord server.

### DISCUSSION THREAD

**Discussion by LyubomirT** [Comon-tech/numQuest#51] [General]
(Sorry if this is not the right place, I couldn't find any other way to contact you properly)

I've noticed that you're not quite active on Discord anymore, and I'd like to help you around with the Comon-tech server as it appears to be a bit dead these days. If it's okay, could you please give the Co-owner role the Administrator permission so I can get control over things in the server that need to be fixed / improved / cleaned up? Thank you. :)

**Comment by C-o-m-o-n** [comment]
> (Sorry if this is not the right place, I couldn't find any other way to contact you properly)
> 
> I've noticed that you're not quite active on Discord anymore, and I'd like to help you around with the Comon-tech server as it appears to be a bit dead these days. If it's okay, could you please give the Co-owner role the Administrator permission so I can get control over things in the server that need to be fixed / improved / cleaned up? Thank you. :)

That's very thoughtful.  I'm sorry for not being around. I'll surely give you the administrator permissions.  The server really needs fixing
:)
